InfrastructureManager/system_manager.py

```diff
@@ -84,9 +84,10 @@ def get_disk_usage(self, secret):
 
     inner_disk_stats_dict = []
     for partition in psutil.disk_partitions(all=True):
-      disk_stats = psutil.disk_usage(partition.mountpoint)
       if partition.mountpoint not in MOUNTPOINT_WHITELIST:
         continue
+
+      disk_stats = psutil.disk_usage(partition.mountpoint)
```

Comment:
Though I absolutely agree that `if..continue` should go before `psutil.disk_usage`, I'd also suggest to remove `all=True` flag, as it tells psutil to extend list of physical devices with other patitions like memory partitions.
[psutil docs says](https://pythonhosted.org/psutil/#psutil.disk_partitions):
> If all parameter is False it tries to distinguish and return physical devices only (e.g. hard disks, cd-rom drives, USB keys) and ignore all others (e.g. memory partitions such as /dev/shm)